spec/filters/elasticsearch_spec.rb

```diff
@@ -4,6 +4,7 @@
 require "logstash/filters/elasticsearch"
 require "logstash/json"
 require "cabin"
+require "manticore"
```

Comment:
Do we really need?

Reply:
Do you have a link to a failed run where you saw this? It looks to me like this is currently running fine https://app.travis-ci.com/github/logstash-plugins/logstash-input-elasticsearch/jobs/635313527

Reply:
@donoghuc the link you have posted is belong to `input-elasticsearch`, this is `filter-elasticsearch` but input-es is also with the same case.
I have seen this issue in the past -https://app.travis-ci.com/github/logstash-plugins/logstash-filter-elasticsearch/jobs/635410594
And it looks like transient. I have retried it.